tests/cross_functional/kcs/test_noobaa_db_backup_and_recovery.py

```diff
@@ -40,6 +41,7 @@ def init_sanity(self):
         """
         self.sanity_helpers = Sanity()
 
+    @rgw
```

Comment:
you are using mcg decorator in the class level, and in function level you are using here rgw.
Which I think that this test will be decorated by both at the end.  `@mcg` from class and `@rgw` from the method decorator.  So my guess is that this test will be executed when user will use `-m mcg` and as well if user will use `m rgw`

Reply:
To mark such tests that use both MCG and RGW with both marks was initial intention since this test technically also uses RGW, but after some discussion it was decided to use either @mcg or @rgw but not both for each test, depending on it's main focus. 

Fixed by removing the @rgw decorators from tests where RGW is not the main focus.